0x05-python-exception task

